zuul.d/projects.yaml

```diff
@@ -55,3 +55,12 @@
     check:
       jobs:
         - packer-master-functional-devstack
+
+- project:
+    name: theopenlab/machine
+    check:
+      jobs:
+        - docker-machine-master-functional-opentelekomcloud
+        - docker-machine-maste-functional-orange
+        - docker-machine-maste-functional-telefonica
+        - docker-machine-maste-functional-huaweicloud
```

Comment:
Like packer, we should add this jobs into periodic pipeline and add `branches: master` option